assets/src/edit-story/components/helpCenter/constants.js

```diff
@@ -41,7 +45,7 @@ export const TIPS = {
   },
   cropElementsWithShapes: {
     title: __('Crop elements using shapes', 'web-stories'),
-    figure: 'images/help-center/media_bg_shape_module_3.webm',
+    figureSrc: 'images/help-center/media_bg_shape_module_3.webm',
```

Comment:
unrelated change but since I touched the file this cleans up the props for another component I touched.